source/variables/transitions.css

```diff
@@ -0,0 +1,10 @@
+:root {
+	/* The amount of time for transitions to take. */
```

Comment:
I'm not sure these are helpful. Having to type `--transition-delay` in place of `0s` is not a win, imo. These defaults were obfuscated by the `transition` mixin in Phoenix and never had to be used directly (much) in practice there. I may be on the wrong side of this, or grumpy, or both, but I can't imagine typing something 1000% longer will gain a lot of traction, even if it is just the name of the property. And I recognize the benefits for having variables of this sort, of course, I'm just not convinced we need these specific variables.